inc/cachify.class.php

```diff
@@ -1124,6 +1086,48 @@ private static function _is_logged_in() {
 		return false;
 	}
 
+	/**
+	 * Register all hooks to flush the total cache
+	 *
+	 * @since   2.4.0
+	 *
+	 * @param   void
+	 * @return  void
+	 */
+	public static function register_flush_cache_hooks(){
+
+		/* Define all default flush cache hooks */
+		$flush_cache_hooks = array(
+			'cachify_flush_cache' 				=> 10,
```

Comment:
Nitpicking: please, fix the white-space formatting here: either align `=>` on every line with proper number of tabs or use single space only before and after every `=>`.